carbonmark/.generated/carbonmark-api-sdk/client.ts

```diff
@@ -19,11 +17,25 @@ export type ApiError<TError> = {
 
 type ParamsObject = Record<string, string | string[]>;
 
-/** Removes all undefined or null values from the object  */
-const definedParams = (obj: Record<string, unknown>) => {
-  const filteredObj = pickBy(obj, notNil);
-  return Object.keys(filteredObj).length > 0 ? filteredObj : null;
-};
+/**
+ * Transforms a request params object into a query string
+ * Handles Arrays
+ */
+export function serializeParams(params: ParamsObject): string {
+  if (params && Object.keys(params).length > 0) {
+    const query: Array<string> = [];
+    Object.keys(params).forEach((key) => {
+      const value = params[key];
+      if (Array.isArray(value))
+        value.forEach((v) => {
+          query.push(`${key}=${encodeURIComponent(v)}`);
+        });
+      else if (value) return query.push(`${key}=${encodeURIComponent(value)}`);
+    });
+    if (query.length) return `?${query.join("&")}`;
+  }
+  return "";
+}
```

Comment:
> I think we can do this a little more comprehensibly:
> 
> Did you find some instances where arrays were being passed?

@0xemc 
I am passing arrays in this PR: https://github.com/KlimaDAO/klimadao/blob/0684e205713a40c87c737ae39b7ce6ae8d0b6e00/carbonmark/components/pages/Project/index.tsx#L61C34-L61C34

Here is the rationale behind that:
- we want (I think) to declare arrays in the fastify schema:  https://github.com/KlimaDAO/klimadao/blob/0684e205713a40c87c737ae39b7ce6ae8d0b6e00/carbonmark-api/src/routes/activities/get.schema.ts#L15 . 
Note that this is not what we do now. We declare strings and we manually split them: https://github.com/KlimaDAO/klimadao/blob/0684e205713a40c87c737ae39b7ce6ae8d0b6e00/carbonmark-api/src/routes/projects/get.ts#L41
- To make query arrays parameters work natively with fastify, they must be formatted this way: ?`country=belgium&country=france` instead of `country=belgium,france` (which is URLSearchParams behaviour).
- Thats is why, I am handling the parameters manually, 

WDYT?